iaso/tests/api/test_orgunits.py

```diff
@@ -934,6 +987,27 @@ def test_edit_org_unit_partial_update(self):
         self.assertEqual(ou.geom.wkt, MultiPolygon(Polygon([(0, 0), (0, 1), (1, 1), (0, 0)])).wkt)
         self.assertEqual(response.data["reference_instances"], [])
 
+    def test_edit_org_unit_partial_update_read_permission(self):
+        """Check tha we can only modify a part of the fille"""
```

Comment:
```suggestion
        """Check that we can only modify a part of the file"""
```

Is this docstring really accurate?

This test seems to ensure that an update is not possible with the read perm.

It looks very much like the test above.

Reply:
@hakifran what do you think about that? Is this test required? Because it seems like a duplicate of the one above?

Reply:
Refactored too, I just pushed the changes